src/lib/SVGMap/ZoomControls.svelte

```diff
@@ -0,0 +1,127 @@
+<script>
+  export let zoomIn;
+  export let zoomOut;
+  export let zoomReset;
+  export let showReset = false;
+  /*
+   * Where the zoom control UI should be positioned.
+   * @type {"bottom-left" | "bottom-right" | "top-left" | "top-right"}
+   */
+  export let controlPosition;
+
+  $: top = controlPosition.includes("top");
+</script>
+
+<div class="zoom-controls {controlPosition}">
+  {#if showReset && top}
```

Comment:
Nice to have would be only showing the reset button when the coords/zoom level do not equal those of the "true" reset. Right now it shows even if you zoom in/back out and don't pan.

In a previous mapbox project, I was logging the coords/zoom level on load (when the map was all the way zoomed out). Then I only showed the reset icon if the coords didn't match. Just sharing as an idea here!

Reply:
that is indeed how this works, the trick is zooming in and out don't always leave you _exactly_ where you started. 

logic here: https://github.com/UrbanInstitute/dataviz-components/pull/89/files#diff-494280830062b5a0c634081adaa111d5dddc505c2c92cc05c7a874316b00b6f7R154

Reply:
actually i needed to get a little more specific with the logic! working better now